lib/Fhp/Segment/DME/HIDMESv2.php

```diff
@@ -2,21 +2,12 @@
 
 namespace Fhp\Segment\DME;
 
-use Fhp\Segment\BaseGeschaeftsvorfallparameter;
-
 /**
  * Segment: Terminierte SEPA-Sammellastschrift einreichen Parameter
  *
  * @link https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Messages_Geschaeftsvorfaelle_2015-08-07_final_version.pdf
- * Section: C.10.3.2.2.1 c)
+ * Section: C.10.3.2.2.2 c)
  */
-class HIDMESv2 extends BaseGeschaeftsvorfallparameter implements HIDXES
+class HIDMESv2 extends HIDMESv1
```

Comment:
It won't really work like this because now there's a `ParameterTerminierteSEPASammellastschriftEinreichenV1` in here (by inheritance) rather than a `ParameterTerminierteSEPASammellastschriftEinreichenV2`.